.github/workflows/create_add_rewards_payload.yaml

```diff
@@ -32,32 +34,55 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: "3.x"
+          python-version: "3.10"
 
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade pip
-          pip install web3
-          pip install bal_addresses
+          pip install git+https://github.com/BalancerMaxis/bal_addresses@0.9.5
 
       - name: Run script
         run: |
-          cd action-scripts
-          python erc20_transfer_payload.py
+          python action-scripts/tx_build_add_reward_token.py
         env:
           TOKEN: ${{ github.event.inputs.token }}
-          DESTINATION: ${{ github.event.inputs.destination }}
-          WHOLE_AMOUNT: ${{ github.event.inputs.whole_amount }}
-          WEI_AMOUNT: ${{ github.event.inputs.wei_amount }}
+          GAUGE: ${{ github.event.inputs.gauge }}
+          DISTRIBUTOR: ${{ github.event.inputs.distributor }}
           MULTISIG: ${{ github.event.inputs.multisig }}
           CHAIN_NAME: ${{ github.event.inputs.chain_name }}
           INFURA_KEY: ${{ secrets.INFURA_KEY }}
           ALCHEMY_KEY: ${{ secrets.ALCHEMY_KEY }}
 
+      - name: Prepare PR title and body
+        run: |
+          echo "PR_TITLE=Automatically generated add_rewards payload (direct incentives) - $TOKEN - $GAUGE" >> $GITHUB_ENV
+          echo "PR_BODY=This pull request was automatically generated by the GitHub Actions workflow. Here are the details of the event that triggered this workflow: - Reward Token: $TOKEN - Gauge: $GAUGE - Distributor: $DISTRIBUTOR - Multisig: $MULTISIG - Chain Name: $CHAIN_NAME" >> $GITHUB_ENV
+        env:
+          TOKEN: ${{ github.event.inputs.token }}
+          GAUGE: ${{ github.event.inputs.gauge }}
+          DISTRIBUTOR: ${{ github.event.inputs.distributor }}
+          MULTISIG: ${{ github.event.inputs.multisig }}
+          CHAIN_NAME: ${{ github.event.inputs.chain_name }}
+
+
+      - name: Generate branch name
+        id: generate-branch-name
+        run: |
+          echo "::set-output name=branch::${{ github.job }}-$(date +'%Y%m%d%H%M%S')"
```

Comment:
what is the reason for using this way of creating the output, vs `outputs:`?

Reply:
I spent a bunch of time fighting with it, some vars you can't pass into the PR action, in that context.  This ended up working.  If you think you can figure out another way to get the job name and the date in the branch: section be my guest.  This step is just using bash to create an output I can read in in Create PR.